specs/lib/obj2gltfSpec.js

```diff
@@ -75,9 +76,10 @@ describe('obj2gltf', function() {
             .then(function() {
                 var args = spy.calls.first().args;
                 var options = args[2];
+                expect(options.basePath).toBeDefined();
+                delete options.basePath; // This will be a random temp directory
```

Comment:
Up to you, but you could use expose `obj2gltf._getTempDirectory` and use a spy to control its value for testing purposes.  This way you know the temp function is being called and set to the expected value. But I'm probably just being picky.  This go go in as-is if you want.